### PR TITLE
Add Shadow Relic

### PR DESCRIPTION
**Pull request by NafenX**
Implemented Shadow Relic (The first time you exert a card each combat, gain 1 Intangible.) along with completely placeholder art.

### CODE REVIEW

File: src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Relic-Strings.json [hunk 1]
```diff
@@ -57,5 +57,12 @@
     "DESCRIPTIONS": [
       "Deal #b3 damage to ALL enemies when you stsjorbsmod:Exert a card."
     ]
+  },
+  "stsjorbsmod:ShadowRelic": {
+    "NAME": "Shadow",
+    "FLAVOR": "A shade of a different CULLor.",
```
Comment (wang429) [inline src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Relic-Strings.json:63]: 👏

**Review by wang429 — CHANGES_REQUESTED** [review]
Thanks for the contribution! I left a few comments inline.

File: src/main/java/stsjorbsmod/relics/ShadowRelic.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+/**
+ * The first time you exert a card each combat, gain 1 Intangible.
+ * TODO: relic stats mod support
+ */
+public class ShadowRelic extends CustomJorbsModRelic {
+    public static final String ID = JorbsMod.makeID(ShadowRelic.class);
+
+    private static final int TURNS = 1;
+    private static boolean usedThisCombat = false;
```
[the remark this answers is not in the document]
Comment (NafenX) [reply src/main/java/stsjorbsmod/relics/ShadowRelic.java:23]: I went off Centennial Puzzle, but can change it if need be

File: src/main/java/stsjorbsmod/relics/ShadowRelic.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import static stsjorbsmod.characters.Cull.Enums.CULL_CARD_COLOR;
+
+/**
+ * The first time you exert a card each combat, gain 1 Intangible.
+ * TODO: relic stats mod support
+ */
+public class ShadowRelic extends CustomJorbsModRelic {
+    public static final String ID = JorbsMod.makeID(ShadowRelic.class);
+
+    private static final int TURNS = 1;
```
[the remark this answers is not in the document]
Comment (NafenX) [reply src/main/java/stsjorbsmod/relics/ShadowRelic.java:22]: NUM_TURNS is used in base game code, I just figured TURNS was simpler.

Maybe NUM_STACKS or INTANGIBLE_STACKS instead?

[the remark this answers is not in the document]
Comment (wang429) [reply src/main/java/stsjorbsmod/relics/ShadowRelic.java]: oh, yeah, `turns` intangible. I guess I like being more explicit. lol

File: src/main/java/stsjorbsmod/relics/ShadowRelic.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (wang429) [reply src/main/java/stsjorbsmod/relics/ShadowRelic.java]: Weird scenario then. I wonder what the devs were thinking with using statics then. I guess I'm fine either way.

File: src/main/java/stsjorbsmod/relics/ShadowRelic.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Override
+    public void onTrigger() {
+        if (AbstractDungeon.getCurrRoom().phase == AbstractRoom.RoomPhase.COMBAT && !usedThisCombat) {
+            flash();
+            pulse = false;
+            addToTop(new RelicAboveCreatureAction(AbstractDungeon.player, this));
+            AbstractCreature p = AbstractDungeon.player;
+            addToBot(new ApplyPowerAction(p, p,new IntangiblePlayerPower(p, TURNS)));
```
Comment (wang429) [inline src/main/java/stsjorbsmod/relics/ShadowRelic.java:43]: Did this compile for you? TURNS isn't defined in this class anymore.

**Review by wang429 — CHANGES_REQUESTED** [review]
Thanks for contributing! Made the one comment in line.

File: src/main/java/stsjorbsmod/relics/ShadowRelic.java (hunk 4 above)
Comment (NafenX) [reply src/main/java/stsjorbsmod/relics/ShadowRelic.java]: that'll teach me to not try to hastily fix things right here in the web editor at about 10 at night :|

**Review by wang429 — APPROVED** [review]
Looks good! Thanks for the contribution!

**Pull request merged**